content/applications/sales/sales/ebay_connector/manage.rst

```diff
@@ -1,50 +1,102 @@
-======================
-How to list a product?
-======================
+=====================
+List products on eBay
+=====================
 
```

Comment:
Is there one sentence that could go in-between the title and the first header as an intro/buffer? Not totally necessary but would be nice